_posts/en/newsletters/2022-03-23-newsletter.md

```diff
@@ -0,0 +1,143 @@
+    resisted by objecting users.
+
+- **Pathfinding algorithm update:** René Pickhardt [posted][pickhardt
+  pathfinding] to the Lightning-Dev mailing list that he'd found a much
+  more computationally effecient approximation for his and Stefan
+  Richter's pathfinding algorithm published last year.  See [Newsletter
+  #163][news163 pp] for earlier discussion about the algorithm.
+
+    Pickhardt's email also suggests ways in which fast payment success
+    could be improved, such as through the implementation of [stuckless
+    payments][news53 stuckless] and by allowing refundable overpayments
+    as proposed in a [recent paper][spear].
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin
+wallets and services.*
+
+- **Coinswap implementation Teleport Transactions announced:**
+  In a recent Bitcoin-Dev mailing list [post][belcher teleport], Chris Belcher
+  announced an alpha [version 0.1][teleport transactions 0.1] of Teleport
+  Transactions that implements the [coinswap][topic coinswap] protocol.
+
+- **JoinMarket adds taproot sends:**
+  [JoinMarket v0.9.5][joinmarket v0.9.5] adds the ability to send to
+  [bech32m][topic bech32] addresses.
+
+- **Mercury Wallet adds RBF support:**
+  Mercury wallet, a wallet for the Mercury [statechain][topic statechains],
+  released [v0.6.5][mercury v0.6.5] including support for [Replace-by-Fee
+  (RBF)][topic rbf] transaction replacements for withdrawals.
+
+- **Hexa Wallet adds Lightning support:**
+  Hexa Wallet, a Bitcoin mobile wallet, added Lightning Network features for LND
+  users running their own node in the [v2.0.71 release][hexa v2.0.71].
+
+- **Sparrow adds BIP47 support:**
+  [Sparrow 1.6.0][sparrow 1.6.0] (and subsequent 1.6.1 and 1.6.2 releases) added
+  support and features for [BIP47][] reusable payment codes and [described the
+  features][sparrow wallet tweet].
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+<!-- FIXME:harding to update Tuesday -->
+
+- HWI FIXME
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Eclair #2203][] adds additional configuration parameters to allow
+  users to specify a different minimum funding for [unannounced
+  channels][topic unannounced channels] than the default used for
+  announced channels.
+
+- [BDK #1311][] adds support for the Short Channel Identifier (SCID)
```

Comment:
```suggestion
- [LDK #1311][] adds support for the Short Channel Identifier (SCID)
```

Reply:
Thanks so much for catching these typos!  I would've been especially embarrassed by them!